include/solvers/vof_scratch_data.h

```diff
@@ -265,6 +326,54 @@ class VOFScratchData : public PhysicsScratchDataBase
       }
   }
 
+
+  /** @brief Reinitializes the content of the scratch regarding the velocity for internal/boundary faces.
+   *  The velocity is inherently assumed to have been solved using a CG scheme.
+   *
+   * @param[in] cell The cell over which the assembly is being carried.
+   *
+   * @param[in] face The face index associated with the cell
```

Comment:
I would keep the same notation as the previous functions, ex:

```suggestion
   * @param[in] face_no The face index associated with the cell
```